bgcval2/functions/AirSeaFluxCO2.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright 2015, Plymouth Marine Laboratory
+#
+# This file is part of the bgc-val library.
+#
+# bgc-val is free software: you can redistribute it and/or modify it
+# under the terms of the Revised Berkeley Software Distribution (BSD) 3-clause license.
+
+# bgc-val is distributed in the hope that it will be useful, but
+# without any warranty; without even the implied warranty of merchantability
+# or fitness for a particular purpose. See the revised BSD license for more details.
+# You should have received a copy of the revised BSD license along with bgc-val.
+# If not, see <http://opensource.org/licenses/BSD-3-Clause>.
+#
+# Address:
+# Plymouth Marine Laboratory
+# Prospect Place, The Hoe
+# Plymouth, PL1 3DH, UK
+#
+# Email:
+# ledm@pml.ac.uk
+#
+"""
+.. module:: TotalAirSeaFluxCO2
+   :platform: Unix
+   :synopsis: This function calculated the total Air Sea FLux for the MEDUSA model in the eORCA grid.
+
+.. moduleauthor:: Lee de Mora <ledm@pml.ac.uk>
+
+"""
+
+import numpy as np
+from bgcval2.bgcvaltools.dataset import dataset
+from bgcval2.functions.get_kwarg_file import get_kwarg_file
+
+global loaded_area_and_mask
+global tmask
+global area
+
+area = 0
+loaded_area_and_mask = False
+
+
+def load_area_and_mask(areafile):
+    if isinstance(areafile, list) and len(areafile)==1:
+        areafile = areafile[0]
+    nc = dataset(areafile, 'r')
+    area = nc.variables['e2t'][:] * nc.variables['e1t'][:]
+    nc.close()
+    loaded_area_and_mask = True
+    return area
+
+
+def calc_total_airseafluxco2(nc, keys, **kwargs):
+    """
+    This function calculated the total Air Sea FLux for the MEDUSA model in the eORCA grid.
+    """
+    areafile = get_kwarg_file(kwargs, 'areafile')
+
+    if not loaded_area_and_mask:
```

Comment:
I think I get what you mean, but we did discuss why we're going this route instead of doing that before. It's so we don't have to load the area file every time this function is called.